addon/controllers/gis-search-form.js

```diff
@@ -117,6 +164,88 @@ export default Ember.Controller.extend({
         fieldName: field
       });
       this.send('doSearch', req);
+    },
+
+    /**
+      Handles click on 'open map' button.
+
+      @method actions.goToMap
+      @param {Object} mapModel Model of the map to transition
+    */
+    goToMap(mapModel) {
+      // may be it should pass event.ctrlKey
+      this.transitToMap(mapModel);
```

Comment:
Now transition can be performed, only in same window, but as i remember we also need ability to, to perform transitions in new window with search form stay opened.
So this ability must be implemented too.